pkg/config/application_ecs.go

```diff
@@ -34,6 +34,10 @@ func (s *ECSApplicationSpec) Validate() error {
 }
 
 type ECSDeploymentInput struct {
+	// The name of network configuration file placing in application directory.
+	VpcConfigurationFile string `json:"vpcConfigurationFile"`
+	// The name of cluster definition file placing in application directory.
+	ClusterDefinitionFile string `json:"clusterDefinitionFile"`
```

Comment:
Looks like this file will only contain the clusterArn, and we will not create the cluster using piped so I think we should pass the clusterArn directly instead. WDYT?

Reply:
I couldn't think of a way to pass the clusterArn directly, so I had to pass it via file...
Do you have any good ideas?😭

Reply:
I mean, treating it as a string and making this field like `clusterArn` is enough, isn't it?
I saw the RunTaskInput only requires ClusterArn 👀 
ref: https://github.com/pipe-cd/pipecd/pull/4084/files/600b7403062cf90e5a806cbb9a92e72981f02594#diff-d72df5132a62e1bc9918e8313b516291c54b87a84acd3686bf5233adf4775cccR156

Reply:
I apologize if I did not understand your intent🙇

`clusterArn` has to be received from outside. because we want to execute the task within the specified cluster.
The only way I could think of to receive `clusterArn` is via file.
Also, The RunTaskInput requires TaskDefinition field.
ref: https://github.com/aws/aws-sdk-go-v2/blob/main/service/ecs/api_op_RunTask.go#L77

Reply:
I mean current implementation of client.RunTask passes whole types.Cluster object but in the function body when you prepare RunTaskInput, only cluster.ClusterArn be used. It means we don't need the whole types.Cluster object anyway, so we should let users pass only input.clusterArn from start instead of focusing them to make a new cluster.yaml which contains almost nothing. 👀

Reply:
Thank you for your detailed explanation😄
I understand. I'll give it a try🚀